rules.json

```diff
@@ -459,6 +459,24 @@
         "code": "function (user, context, callback) {\n  if (!user.app_metadata || !user.app_metadata.authorization ||\n  !Array.isArray(user.app_metadata.authorization.groups)) {\n    return callback(null, user, context);\n  }\n\n  var groups = user.app_metadata.authorization.groups;\n  var GROUPS_WITH_MFA = {\n    // Add groups that need MFA here\n    // Example\n    // 'admins': true,\n  };\n\n  var needsMFA = !!groups.find(function(group) {\n    return GROUPS_WITH_MFA[group];\n  });\n\n  if (needsMFA){\n      context.multifactor = {\n        provider: 'guardian', //required\n\n        ignoreCookie: true, // optional. Force Auth0 MFA everytime this rule runs. Defaults to false. if accepted by users the cookie lasts for 30 days (this cannot be changed)\n      };\n  }\n\n  callback(null, user, context);\n}"
       },
       {
+        "title": "Force multifactor when IP outside an expected range",
+        "categories": [
+          "multifactor",
+          "guardian"
+        ],
+        "description": "<p>This rule is used to trigger multifactor authentication when the requesting IP is from outside the corporate IP range.</p>\n",
+        "code": "function (user, context, callback) {\n  var ipaddr = require('ipaddr.js');\n  var corp_network = '192.168.1.134/26';\n\n  var current_ip = ipaddr.parse(c  ontext.request.ip); \n  if(!current_ip.match(ipaddr.parseCIDR(corp_network))){\n    context.multifactor = {\n        provider: 'guardian',\n        ignoreCookie: true, \n        };  \n  }\n  \n  callback(null, user, context);\n}\n"
```

Comment:
@tjanczuk @topherMarie fyi, the `rules.json` is autogenerated from this script: https://github.com/auth0/rules/blob/master/build

and it runs with every `npm version {version}` command as shown here: https://github.com/auth0/rules/blob/master/package.json#L7

So there is no need to submit the `rules.json` file with every commit.